cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -19,151 +19,43 @@ metadata:
   labels:
     {{- include "cost-analyzer.commonLabels" . | nindent 4 }}
 rules:
-  - apiGroups:
-      - kubecost.com
-    resources:
-      - turndownschedules
-      - turndownschedules/status
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - patch
-      - update
-      - delete
-  - apiGroups:
-      - ''
-      - events.k8s.io
-    resources:
-      - events
-    verbs:
-      - create
-      - patch
-      - update
-  - apiGroups:
-      - ''
-    resources:
-      - deployments
-      - nodes
-      - pods
-      - resourcequotas
-      - replicationcontrollers
-      - limitranges
-      - pods/eviction
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - patch
-      - update
-      - delete
-  - apiGroups:
-      - ''
-    resources:
-      - configmaps
-    verbs:
-      - get
-      - list
-      - watch
-      - update
-      - patch
-      - delete
-  - apiGroups:
-      - ''
-    resources:
-      - configmaps
-      - namespaces
-      - persistentvolumeclaims
-      - persistentvolumes
-      - endpoints
-      - events
-      - services
-    verbs:
-      - get 
-      - list
-      - watch
-  - apiGroups:
-      - ''
-    resources:
-      - configmaps
-    resourceNames:
-      - 'cluster-controller-nsturndown-config'
-    verbs:
-      - get
-      - create
-      - update
-  - apiGroups:
-      - apps
-    resources:
-      - statefulsets
-      - deployments
-      - daemonsets
-      - replicasets
-    verbs:
-      - get 
-      - list
-      - watch
-      - create
-      - patch
-      - update
-      - delete
-  - apiGroups:
-      - batch
-    resources:
-      - cronjobs
-      - jobs
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - patch
-      - update
-      - delete
-  - apiGroups:
-      - autoscaling
-    resources:
-      - horizontalpodautoscalers
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - policy
-    resources:
-      - poddisruptionbudgets
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups: 
-      - storage.k8s.io
-    resources: 
-      - storageclasses
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - events.k8s.io
-    resources:
-      - events
-    verbs:
-      - get
-      - list
-      - watch
+  - apiGroups: ['kubecost.com']
```

Comment:
@mittal-ishaan , the changes LGTM, can you rebase though on @Sean-Holcomb 's large refactor PR? trying to minimize merge conflicts there